ros2topic/ros2topic/api/__init__.py

```diff
@@ -114,6 +114,17 @@ def set_msg_fields(msg, values):
         except ValueError as e:
             raise SetFieldError(field_name, e)
         try:
+            f_type = msg.get_slot_types()['_' + field_name]
```

Comment:
This looks pretty weird see https://github.com/ros2/rosidl_python/pull/30#issue-253692496 for more context and suggestion for stripping the leading underscores